tests/routes_tests/test_audit_boards.py

```diff
@@ -618,11 +618,24 @@ def run_audit_board_flow(jurisdiction_id: str, audit_board_id: str):
         [{"name": "Audit Board #1"}],
     )
     assert_ok(rv)
+
     rv = client.get(
         f"/election/{election_id}/jurisdiction/{jurisdiction_ids[1]}/round/{round_1_id}/audit-board"
     )
     audit_board = json.loads(rv.data)["auditBoards"][0]
 
+    # Create another audit board that doesn't have any ballots assigned. Even
+    # though this audit board doesn't sign off, it shouldn't stop the round
+    # from being completed.
+    audit_board_without_ballots = AuditBoard(
+        id="audit-board-without-ballots",
```

Comment:
I'm pretty sure this won't work the second time around, since this ID has to be unique throughout the database. I'd just use a UUID here. The name is already good.

Reply:
Shouldn't matter because it's a test, right? But I'll fix it to make it obvious it's not an issue

Reply:
ah yes, you're right, a lot less problematic than I thought. I got a little bit confused about the purpose of this audit board.

Yes, better to fix it I think, just to not make things fail if this runs twice, but not the problem I thought it was :)